ziti/ziti.go

```diff
@@ -57,6 +57,7 @@ type Context interface {
 	GetServiceId(serviceName string) (string, bool, error)
 	GetServices() ([]edge.Service, error)
 	GetService(serviceName string) (*edge.Service, bool)
+	SetAppInfo(appId, appVersion string)
```

Comment:
I don't think these should be per-context values, the app would be the same for all ziti contexts

Reply:
Ok

Reply:
Fixed, let me know.